hooks/post/30-upgrade.rb

```diff
@@ -176,39 +72,7 @@ def fail_and_exit(message)
 if app_value(:upgrade)
   if [0, 2].include?(@kafo.exit_code)
     upgrade_step :restart_services, :run_always => true
```

Comment:
IIRC we stop all services before upgrading and the installer should start all services it needs (or restart if anything changed). That makes me wonder if this is really needed.

Reply:
@ehelms your thoughts on this?

Reply:
I tend to agree we should be able to drop this

Reply:
any idea why we check for a return code of 2?